201-vm-winrm-windows/winrmconf.cmd

```diff
@@ -0,0 +1,2 @@
+call winrm set winrm/config/service/auth @{Basic="true"}
```

Comment:
This file is never used in the deployment...

Reply:
Yes, currently the file is not required for the template. But, we were referring this file in our on-premise product. I will clean this file soon once all the dependencies are resolved.